config/secrets.yml

```diff
@@ -13,10 +13,32 @@
 development:
   secret_key_base: 74aa64763bf392a5f975f2bece3c13cbcd85687f920abe0427b2ebef0a3411ba3dcd87cdc91c4cdbe3105bf49c9486926da2f745dc1a7889ad251032b9212f4a
 
+  twitter_app_id: N01qhL9a5erk5Zjga0GIi9I2K
+  twitter_app_secret: fZSN4NHJIzkKXLkz88sxi9z6EwIVyDWxiRhgqd03bKBFLr4pVx
+
+  facebook_app_id: <%= ENV['FACEBOOK_APP_ID'] %>
+  facebook_app_secret: <%= ENV['FACEBOOK_APP_SECRET'] %>
+
+  aws_access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
+  aws_secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
+  aws_s3_bucket: <%= ENV['AWS_S3_BUCKET'] %>
+
 test:
   secret_key_base: 877762d957de5d685c73abf39173618509d34ba2b80ec83e1459f19c58ada5a68d8a2439a5b77863e509c5de6093a1c8b965a7ec55ad1fdcedbaa758cc52f270
 
+  twitter_app_id: N01qhL9a5erk5Zjga0GIi9I2K
+  twitter_app_secret: fZSN4NHJIzkKXLkz88sxi9z6EwIVyDWxiRhgqd03bKBFLr4pVx
+
+  facebook_app_id: <%= ENV['FACEBOOK_APP_ID'] %>
+  facebook_app_secret: <%= ENV['FACEBOOK_APP_SECRET'] %>
+
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
```

Comment:
Good catch.
To simplify this yml file, I think we could use

test:
   << *default
production:
  << *default
as it is in database.yml